ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFContactCollectionView)
+open class ContactCollectionView: UICollectionView {
+    struct Constants {
+        static let sideInset: CGFloat = 16.0
+        static let extraSmallContentContactHeight: CGFloat = 115.0
+        static let smallContentContactHeight: CGFloat = 117.0
+        static let mediumContentContactHeight: CGFloat = 118.0
+        static let largeContentContactHeight: CGFloat = 121.0
+        static let extraLargeContentContactHeight: CGFloat = 125.0
+        static let extraExtraLargeContentContactHeight: CGFloat = 129.0
+        static let extraExtraExtraLargeContentContactHeight: CGFloat = 135.0
+    }
+
+    let layout: ContactCollectionViewLayout
+    @objc public weak var contactCollectionDelegate: UICollectionViewDelegate?
+
+    @objc public var contactList: [PersonaData] = [] {
+        didSet {
+            if (oldValue.count == 0 && contactList.count > 0) || (oldValue.count > 0 && contactList.count == 0) {
+                updateHeightConstraint()
+            }
+        }
+    }
+
+    @objc public init() {
+        layout = ContactCollectionViewLayout()
+        layout.scrollDirection = .horizontal
+        super.init(frame: .zero, collectionViewLayout: layout)
+        configureCollectionView()
+        updateHeightConstraint()
+        register(ContactCollectionViewCell.self, forCellWithReuseIdentifier: ContactCollectionViewCell.identifier)
+        NotificationCenter.default.addObserver(self, selector: #selector(setNewHeightConstraint), name: UIContentSizeCategory.didChangeNotification, object: nil)
+    }
+
+    @objc public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    var heightConstraint: NSLayoutConstraint?
+    var widthConstraint: NSLayoutConstraint?
+
+    private func updateHeightConstraint() {
+        if heightConstraint == nil {
+            heightConstraint = heightAnchor.constraint(equalToConstant: 0.0)
+            heightConstraint!.isActive = true
+            return
+        }
+
+        var constant: CGFloat = 0.0
+        if contactList.count > 0 {
+            constant = categoryHeight()
+        }
+
+        heightConstraint!.constant = constant
+    }
+
+    private func configureCollectionView() {
+        translatesAutoresizingMaskIntoConstraints = false
+        showsHorizontalScrollIndicator = false
+        showsVerticalScrollIndicator = false
+        backgroundColor = Colors.surfacePrimary
+        dataSource = self
+        delegate = self
+        contentInset = UIEdgeInsets(top: 0, left: Constants.sideInset, bottom: 0, right: 0)
+
+        if #available(iOS 13, *) {
+            addInteraction(UILargeContentViewerInteraction())
```

Comment:
we are supporting largecontentviewer? didn't the title size increase?

Reply:
The title size does indeed increase when we change the `ContentSizeCategory`. I added the `LargeContentViewer` due to a suggestion I got from Tom. I can discuss this further with Kaan to see if this is something we would like to support.

Reply:
From emails exchanged with Kaan, it has been decided that we don't need large content viewer for the Contact Collection. The relevant code for it will be moved.